### PR TITLE
feat : Single video

### PR DESCRIPTION
**Pull request by arpit01923**
Description-

- videoPlayer page functionality should be added
- contains a api data
- add components navbar, main, footer
- click on single video will link you to videoPlayer of that video

File to check-

- src/pages/video-listing/video-card.jsx
- src/pages/video-player/main.jsx

Preview-

https://user-images.githubusercontent.com/86866251/164890053-dc507795-7ccb-405a-91bd-2b3d6ea2dbf8.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,10 +21,12 @@ function App() {
       <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
-        <Route path="/videoListing" element={<VideoListing />} >
+        <Route path="/videoListing" element={<VideoListing />}>
           <Route path=":category" element={<VideoListing />} />
         </Route>
-        <Route path="/videoPlayer" element={<VideoPlayer />} />
+        <Route path="/videoPlayer" element={<VideoPlayer />}>
+          <Route path=":videoId" element={<VideoPlayer />} />
```
Comment (jeetbhalani15) [inline src/App.js:28]: should have Added /videoplayer/:videoId in one route itself

**Review by jeetbhalani15 — APPROVED** [review]
Made this changes rest all look good

File: src/pages/video-listing/video-card.jsx [hunk 2]
```diff
@@ -11,7 +13,9 @@ export const VideoCard = ({
   return (
     <section className="card badge-card" key={id}>
       <button className="timing">{time}</button>
-      <img className="img" src={poster} alt="card-image" />
+      <Link to={`/videoPlayer/${id}`}>
+        <img className="img" src={poster} alt="card-image" />
```
Comment (Aditi0505) [inline src/pages/video-listing/video-card.jsx]: Change alt to card-poster

File: src/pages/video-player/main.jsx [hunk 3]
```diff
@@ -12,27 +20,27 @@ export const Main = () => {
...
-          <h2 className="margin-top-1 title">title</h2>
+          <h2 className="margin-top-1 title">{filterVideo.title}</h2>
 
           <div className="margin-top-2 footer">
             <span className="sub-container1">
               <img
                 className="round sm"
-                src="https://media.istockphoto.com/photos/businessman-silhouette-as-avatar-or-default-profile-picture-picture-id476085198?b=1&k=20&m=476085198&s=170667a&w=0&h=Ct4e1kIOdCOrEgvsQg4A1qeuQv944pPFORUQcaGw4oI="
+                src={filterVideo.image}
                 alt="avatar-image"
```
Comment (Aditi0505) [inline src/pages/video-player/main.jsx]: Change this to video-avatar

**Review by Aditi0505 — APPROVED** [review]
Left few suggestion.
Also, please look into @jeetbhalani15 suggestion.

1. The code is clean and well written
2. Good use of semantic html

File: src/App.js (hunk 1 above)
Comment (arpit01923) [reply src/App.js:28]: structure looks good

**Pull request merged**